src/guess.js

```diff
@@ -0,0 +1,65 @@
+import compareNumbers from './compareNumbers.js';
+
+const numberInput = document.getElementById('number-input');
+const guessButton = document.getElementById('guess-button');
+const high = document.getElementById('high');
+const low = document.getElementById('low');
+const win = document.getElementById('win');
+const lose = document.getElementById('lose');
+const tries = document.getElementById('tries-left');
+// random number generator - credit to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+const actual = Math.floor(Math.random() * 20 + 1);
+
+let triesLeft = 5;
+
+//UI work functions
```

Comment:
In the future, please delete comments like these before submitting! Your credit for the generator is totally fine (thank you for including that). It's awesome to see that you're organizing your thoughts and outlining things as you work through them, but starting next week I'll have to take credit off for comments left in code.